src/prefs/ModulePrefs.cpp

```diff
@@ -109,21 +109,11 @@ void ModulePrefs::PopulateOrExchange(ShuttleGui & S)
    S.SetBorder(2);
    S.StartScroller();
 
-   S.StartStatic( {} );
+    S.StartStatic( {} );
    {
       S.AddFixedText(XO(
-"These are experimental modules. Enable them only if you've read the Audacity Manual\nand know what you are doing.") );
-      S.AddFixedText(XO(
-/* i18n-hint preserve the leading spaces */
-"  'Ask' means Audacity will ask if you want to load the module each time it starts.") );
-      S.AddFixedText(XO(
-/* i18n-hint preserve the leading spaces */
-"  'Failed' means Audacity thinks the module is broken and won't run it.") );
-      S.AddFixedText(XO(
-/* i18n-hint preserve the leading spaces */
-"  'New' means no choice has been made yet.") );
-      S.AddFixedText(XO(
-"Changes to these settings only take effect when Audacity starts up."));
+"Modules are optional components of Audacity which enable some functionality, such as importing and exporting. \nIt is generally not necessary to change these settings.\n") );
```

Comment:
There is a subtlety in English about "that" versus "which".

The first identifies or distinguishes something or someone, while the latter adds information.  At least I think you will find that recommendation in style guides, though not all natives do it consistently.

I would write "that" here.